ydb/core/tx/datashard/datashard.h

```diff
@@ -1383,6 +1383,25 @@ struct TEvDataShard {
         }
     };
 
+    struct TEvObjectStorageListingRequest
```

Comment:
Could you make some tests in `ydb/core/tx/datashard` folder? Without `ydb/core/client`.